cds/modules/cds_iiif/ext.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""CDS IIIF extension initialization."""
+
+from __future__ import absolute_import, print_function
+
+from flask_iiif import IIIF
+from flask_iiif.cache import simple
+from flask_restful import Api
+
+from .utils import image_opener
+
+
+class CDSIIIF(object):
+    """CDS IIIF extension."""
+
+    def __init__(self, app=None):
+        """Extension initialization."""
+        if app:
+            self.init_app(app)
+
+    def init_app(self, app):
+        """Flask application initialization."""
+        api = Api(app=app)
+        app.config['IIIF_CACHE_HANDLER'] = simple.ImageSimpleCache()
+        ext = IIIF()
+        ext.init_app(app)
+        ext.uuid_to_image_opener_handler(image_opener)
+        ext.init_restful(api, prefix='/iiif/')
```

Comment:
I think to follow the other modules this should got under `/api/iiif` as the rest of the REST APIs, @drjova?